### DISCUSSION TITLE
Docker Instaltion

### DISCUSSION THREAD

**Discussion by Kenny2k8** [jonaswinkler/paperless-ng#1696] [Support]
Hello,

I installed paperless-ng on my Synology NAS with Docker

So it runs but on my local page i cant log in to paperless-ng

Did paperless has after the first install over docker a username and password to login?

kind regards

**Comment by devPeete** [comment]
You have to run 
$ docker-compose run --rm webserver createsuperuser
this will prompt a username and a password. (See https://paperless-ng.readthedocs.io/en/latest/setup.html#install-paperless-from-docker-hub step 7)

However , I suggest you to Switch to Paperless-ngx, as this fork might be abandoned (See https://github.com/paperless-ngx/paperless-ngx). By now you only have to change the docker image.
